kbs/src/attestation/backend.rs

```diff
@@ -276,11 +279,13 @@ impl AttestationService {
 
         let attestation_str =
             serde_json::to_string(&attestation).context("serialize attestation failed")?;
-        let token = self
-            .inner
-            .verify(tee, &nonce, &attestation_str)
-            .await
-            .context("verify TEE evidence failed")?;
+        let token = match self.inner.verify(tee, &nonce, &attestation_str).await {
+            Ok(token) => token,
+            Err(err) => {
+                ATTESTATION_FAILURES.inc();
+                return Err(err).context("verify TEE evidence failed");
+            }
+        };
```

Comment:
There is a way for logging but I am not sure if the compiler will cry for this
```rust
self.inner
    .verify(tee, &nonce, &attestation_str)
    .await
    .inspect(|_| ATTESTATION_FAILURES.inc())
    .context("verify TEE evidence failed")?;
```

Reply:
That's a great point, the compiler doesn't mind it :-) and it actually works, too (provided I use `inspect_err()`, not `inspect()` which is only called if value is `Ok()`) - thanks!